public/js/duplicates.js

```diff
@@ -0,0 +1,191 @@
+    rows.each(function () {
+        var row = $(this);
+        if (rowToExclude && row[0] !== rowToExclude[0]) {
+            row.find('.form-check-input').prop('checked', true);
+        }
+    });
+}
+
+Duplicates.conditionChange = function (event) {
+    var option = $(event.target).val();
+
+    // Clear current selection
+    $('.form-check-input').prop('checked', false);
+
+    // Early return if none should be selected
+    if (option === 'none') {
+        return;
+    }
+
+    $('.duplicate-group').each((_, group) => {
+        // Find all rows of a group
+        var groupRow = $(group);
+        var rowsInGroup = groupRow.add(groupRow.nextUntil('.separator'));
+
+        // Compare rows in group according to selected option
+        switch (option) {
+            case 'less-tags':
+                Duplicates.compareDuplicates(rowsInGroup, "tag-count", "integer");
+                break;
+            case 'less-size':
+                Duplicates.compareDuplicates(rowsInGroup, "file-size", "float");
+                break;
+            case 'less-pages':
+                Duplicates.compareDuplicates(rowsInGroup, "page-count", "integer");
+                break;
+            case 'not-old':
+                Duplicates.compareDuplicates(rowsInGroup, "date-added", "date");
+                break;
+            case 'not-young':
+                Duplicates.compareDuplicates(rowsInGroup, "date-added", "date", "asc");
+                break;
+        };
+    });
+};
+
+Duplicates.deleteArchive = function (event) {
+    LRR.showPopUp({
+        text: "Are you sure you want to delete this archive?",
+        icon: "warning",
+        showCancelButton: true,
+        focusConfirm: false,
+        confirmButtonText: "Yes, delete it!",
+        reverseButtons: true,
+        confirmButtonColor: "#d33",
+    }).then((result) => {
+        if (result.isConfirmed) {
+            archiveid = $(event.currentTarget).attr('data-id');
+            Server.deleteArchive(archiveid, () => { Duplicates.dt.row($(event.currentTarget).parents('tr')).remove().draw()});
+        }
+    });
+};
+
+Duplicates.deleteArchives = function () {
+    LRR.showPopUp({
+        text: "Are you sure you want to delete all selected archives?",
+        icon: "warning",
+        showCancelButton: true,
+        focusConfirm: false,
+        confirmButtonText: "Yes, delete all!",
+        reverseButtons: true,
+        confirmButtonColor: "#d33",
+    }).then((result) => {
+        if (result.isConfirmed) {
+            $("table tbody tr").each(function () {
+                const row = $(this);
+                const isChecked = row.find(".form-check-input").is(":checked");
+                const dataId = row.find(".delete-archive").attr("data-id");
+
+                if (isChecked && dataId) {
+                    Server.deleteArchive(dataId, () => { Duplicates.dt.row(row).remove().draw() });
```

Comment:
As mentioned in the previous comments, it'd be fine to add an optional boolean to `deleteArchive` to make it not emit the success toast.  

(The success toast doesn't really makes sense here as well since it doesn't actually reload the page, too)